tests/unit/test_utils.py

```diff
@@ -1,9 +1,69 @@
+"""Test utils"""
+
+import tempfile
+import os
+from unittest import mock
 from autolabel import utils
 from rich.console import Console
 
 console = Console()
 
 
+def generate_tempfile_with_content(input_url: str) -> None:
+    """Generate a Temporary file with dummy content"""
+    with tempfile.NamedTemporaryFile(dir="./", delete=False) as tmp_file:
+        file_name = os.path.basename(input_url)
+        os.rename(tmp_file.name, file_name)
+        tmp_file.write(f"{input_url}".encode("utf-8"))
+        tmp_file.flush()
+
+
+@mock.patch("wget.download", side_effect=generate_tempfile_with_content)
```

Comment:
Hi! We are using `mocker` pytest fixture for mocking function calls in autolabel tests currently. 

Here's a few examples: https://github.com/refuel-ai/autolabel/blob/main/tests/unit/llm_test.py#L19

Can we change these tests to instead use mocker?

Reply:
Done